pages/login.tsx

```diff
@@ -158,6 +163,11 @@ const LoginPage: React.FC & WithLayout = () => {
       await loginUser({ variables: values })
     } catch {} // catch error that's thrown by default from mutation
   }
+
+  //checks if user is already logged in
+  const session = session_data?.session?.user
```

Comment:
I discussed the currently used approach with someone else on the team prior to doing it this way, but I am open to your suggestions. Can you point towards an example of how lodash can be used here? Im not too familiar with it.

Reply:
I think he means that instead of using `session_data?.session?.user`, we could use `get(session_data, 'session.user')` to stay consistent with what is in the file.

To read more about `get` method: https://lodash.com/docs/4.17.15#get

Reply:
Good to know about lodash. I updated the file to get the session data via lodash.